File: 392-is-subsequence/392-is-subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if(len(s)==0):
            return True
        i=0
        j=0
        while j<len(t):
            if(i==len(s)):
                return True
            if s[i]==t[j]:
                i+=1
            j+=1
        if i==len(s):
            return True
        else:
            return False

File: 392-is-subsequence/392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int i=0, j=0;
        while(j<t.size()){
            if(s[i]==t[j]){
                i++;
            }
            j++;
        }
        return i==s.size()? true: false;
    }
};